mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -643,6 +643,48 @@
     <!-- Preference for autoplay -->
     <string name="preference_autoplay">Autoplay</string>
 
+    <!-- Heading for the instructions to allow a permission -->
+    <string name="phone_feature_blocked_intro">To allow it:</string>
+
+    <!-- First step for the allowing a permission -->
+    <string name="phone_feature_blocked_step_settings">1. Go to Android Settings</string>
```

Comment:
the string should be formatted with `CDATA` as it is for the other points below.

Reply:
No it shouldn't be with CDATA because it doesn't contains any bold characters ("<b>")   or any dynamic string ("%1$s") @Delphine . It's only a text .

Reply:
Woops, totally missed that. You're correct